README.md

```diff
@@ -16,16 +16,18 @@
 **Table of Contents**
 
 - [Installation](#installation)
-- [Usage](#usage)
-- [Library features](#library-features)
-    - [New Relic configuration](#new-relic-configuration)
-    - [Rack middleware](#rack-middleware)
-    - [Database configuration](#database-configuration)
-    - [Sidekiq](#sidekiq)
-    - [HireFire Workers](#hirefire-workers)
-    - [Logging](#logging)
-    - [Google Oauth](#google-oauth)
+- [Configuration and usage](#configuration-and-usage)
```

Comment:
Absolutely! I was a bit confused by this table of content myself, partly because it changed while I was working on this PR. I'll rename **"Configuration and usage"** to **"Library usage"** 👍